### PR TITLE
Added Yaml configs for training.

### PR DESCRIPTION
**Pull request by wiktorfilipiuk**

### CODE REVIEW

**Comment by wiktorfilipiuk** [comment]
Large chunks of commented arguments are still there in order to know how to write docstring later on. Please don't comment these parts for now :)

File: assets/sanity_check_data/train_config_sanity_gloss.yml [hunk 1]
```diff
@@ -40,11 +64,11 @@ heads:
...
             num_class: 37
             loss_weight: 0
-            
+
 freeze_scheduler:
     model_params:
-        feature_extractor: [True, False]
-        transformer: [False, True]
+        feature_extractor": [True, False]
+        transformer": [False, True]
```
Comment (AgaMiko) [inline assets/sanity_check_data/train_config_sanity_gloss.yml:71]: `"` accidentally returned :)

File: assets/sanity_check_data/train_config_sanity_hamnosys.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            num_class: 5
+            loss_weight: 0
+        hand_location_y:
+            num_class: 37
+            loss_weight: 0
+
+freeze_scheduler:
+    model_params:
+        feature_extractor": [True, False]
+        transformer": [False, True]
```
Comment (AgaMiko) [inline assets/sanity_check_data/train_config_sanity_hamnosys.yml:71]: here `"` also -- to be removed

**Review by AgaMiko — CHANGES_REQUESTED** [review]
Looks good to me! Great job! 
Please remember to remove `"` from the freeze scheduler config.

File: models/model.py [hunk 3]
```diff
@@ -30,63 +32,68 @@ class GlossTranslationModel(pl.LightningModule):
...
+                    - "dropout_rate" (float)
+            train_parameters (Dict): Dict containing parameters parameterizing the training process.
+                [Warning] Must containt fields:
+                    - "num_segments" (int)
+                    - "lr" (float)
+                    - "multiply_lr_step" (float)
+                    - "warmup_steps" (float)
+                    - "classification_mode" (str)
+            heads (Dict): Dict containg information describing structure of output heads for specific tasks (gloss/hamnosys).
+            freeze_scheduler (Dict): Dict containing information describing feature_extractor & transformer freezingz/unfreezing process.
```
Comment (alicjak1519) [inline models/model.py:69]: typos:
- `containt` in 54, 62
-  `containg` in 66
- `freezingz` in 67

The same are in the model_for_pretraining.py ;)

File: train.py [hunk 4]
```diff
@@ -20,109 +20,27 @@
...
-        action="store_true",
-        default=False,
-        help="flag to enable reading landmarks annotations",
-    )
     parser.add_argument(
         "--model_config_path",
         type=str,
-        default='train_config_default.yml',
+        default='train_config.yml',
         help="path to .yaml config file specyfing hyperparameters of different model sections."
```
Comment (alicjak1519) [inline train.py:27]: `specyfing` --> `specifying`

**Review by alicjak1519 — COMMENTED** [review]
Your docstrings look great, they explain each parameter 😍

In the model classes implementations, there is a Dict typehint, but there is no "from typing import Dict"

**Review by mariaogryczak — APPROVED** [review]

**Comment by mariaogryczak** [comment]
with changes proposed by @AgaMiko and @alicjak1519 I think it's ok :)

**Review by AgaMiko — APPROVED** [review]

**Pull request closed without merging**